public/index.html

```diff
@@ -433,6 +376,23 @@ <h3 class="text-2xl font-bold mb-2">The conference of smiles | Made with love by
             </section>
         </main>
     </div>
+
+    <script>
+        function duplicateSponsorImages() {
+            const sponsorWrapper = document.getElementById('sponsor-wrapper');
+            if (!sponsorWrapper) return;
+
+            const originalLinks = sponsorWrapper.querySelectorAll('a:not(.duplicate)');
+
+            originalLinks.forEach(link => {
+                const duplicate = link.cloneNode(true);
+                duplicate.classList.add('duplicate');
+                sponsorWrapper.appendChild(duplicate);
+            });
+        }
+
+        document.addEventListener('DOMContentLoaded', duplicateSponsorImages);
+    </script>
 </body>
 
 </html>
```

Comment:
This and the next file are missing their final newline, which some Unix tools struggle with.